financial-connections-lite/src/main/java/com/stripe/android/financialconnections/lite/FinancialConnectionsSheetLiteActivity.kt

```diff
@@ -60,6 +63,26 @@ internal class FinancialConnectionsSheetLiteActivity : ComponentActivity(R.layou
         }
     }
 
+    private fun setupBackButtonHandling() {
+        onBackPressedDispatcher.addCallback(this, object : OnBackPressedCallback(true) {
```

Comment:
Nit: We can use a trailing lambda here.

Reply:
we can't, given `OnBackPressedCallback` accepts a parameter